tests/TestHelpers/HttpRequestHelper.php

```diff
@@ -41,7 +41,8 @@
  */
 class HttpRequestHelper {
 	public const HTTP_TOO_EARLY = 425;
-
+	public const HTTP_CONFLICT = 409;
+	public const MAKE_FOLDER_API = "https://localhost:9200/remote.php/dav/files/";
```

Comment:
ocis url will be different in different environment

Reply:
I think there should be a function `getBaseurl` in some helper filer that you can use to construct this